cli/command/manifest/inspect_test.go

```diff
@@ -0,0 +1,64 @@
+package manifest
+
+import (
+	"bytes"
+	"io"
+	"io/ioutil"
+	"strings"
+	"testing"
+
+	"github.com/docker/cli/cli/internal/test"
+	"github.com/docker/docker/pkg/testutil"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewInspectCommand(t *testing.T) {
+	testCases := []struct {
+		name		string
+		args		[]string
+		expectedError	string
+	}{
+		{
+			name:		"wrong-args",
+			args:		[]string{},
+			expectedError:	"\"inspect\" requires exactly 1 argument(s).",
+		},
+	}
+	for _, tc := range testCases {
+		buf := new(bytes.Buffer)
+		cmd := newInspectCommand(test.NewFakeCli(&fakeClient{}, buf))
+		cmd.SetOutput(ioutil.Discard)
+		cmd.SetArgs(tc.args)
+		testutil.ErrorContains(t, cmd.Execute(), tc.expectedError)
+
+	}
+}
+
+// use testdata and golden for this stuff
+func TestNewInspectCreateSuccess(t *testing.T) {
+	testCases := []struct{
+		name		string
+		args		[]string
+	}{
+		{
+			name:	"simple",
+			args:	[]string{"busybox"},
```

Comment:
Yup, you can use anything here. These are all unit tests, so they never hit a real API. The client is a fake that returns whatever response you want it to return.